assets/src/apps/delivery/layouts/deck/DeckLayoutView.tsx

```diff
@@ -199,9 +199,6 @@ const DeckLayoutView: React.FC<LayoutProps> = ({ pageTitle, pageContent, preview
       setActivityClasses(
         [...new Set([...defaultClasses, ...customClasses])].map((str) => str.trim()),
       );
-    } else if (currentActivity?.content?.partsLayout) {
```

Comment:
We don't need this entire section. It was done to add vft class based on the Id but now we have getCustomClassAncestry() that does all the things.

I already removed below code but looks like entire if block needs to be removed

```
let hasVft = false;
      currentActivityTree.forEach((activity) => {
        if (!hasVft) {
          hasVft = activity?.content?.partsLayout.some((part: any) => part.id === 'vft');
        }
      });
```